tests/unit-tests/blocks/test-class-sensei-continue-course-block.php

```diff
@@ -92,4 +92,22 @@ public function testRender_EnrolledAndCourseNotCompleted_ReturnsModifiedBlockCon
 
 		$this->assertRegExp( '|<a href="http://example.org/\?course=continue-course-block".*>Continue</a>|', $result );
 	}
+
+	public function testRender_EnrolledAndStartedLesson_ReturnsModifiedBlockContentWithLessonUrl() {
```

Comment:
I think we could add more test cases here to ensure we have one for every possible `if` / `else` branch in `get_target_page_post_id_for_continue_url`.